### DISCUSSION TITLE
Recent Saved Search List

### DISCUSSION THREAD

**Discussion by andruwaltringer** [burkybang/SuiteAdvanced-Feedback#70] [Ideas]
Have a drop down list on the Home Page similar to the Recent Records list but for Saved Searches displayed in descending order from the last time the search was ran. Just a thought...

**Comment by burkybang** [comment]
I added this in version 26.6.0

![image](https://user-images.githubusercontent.com/927886/235954151-eccf2f9e-d4be-4901-988b-33f5c59f3f2f.png)
